lib/mock_collector/collector.rb

```diff
@@ -0,0 +1,221 @@
+require "topological_inventory-ingress_api-client"
+
+require "config"
+require "concurrent"
+require "mock_collector/parser"
+require "mock_collector/server"
+
+module MockCollector
+  class Collector
+    def initialize(source, _config, default_limit: 100, poll_time: 30)
+      # self.connection_manager = MockCollector::Server.new
+      self.collector_threads  = Concurrent::Map.new
+      self.finished           = Concurrent::AtomicBoolean.new(false)
+      self.limits             = Hash.new(default_limit)
+      self.log                = Logger.new(STDOUT)
+      self.openshift_host     = nil # openshift_host
+      self.openshift_port     = nil # openshift_port
+      self.openshift_token    = nil # openshift_token
```

Comment:
We probably don't need these three

Reply:
@agrare yeah, I forgot to add WIP before I went home yesterday :)

Reply:
Ah okay no worries!